Sources/SQLKit/Query/SQLDropTable.swift

```diff
@@ -26,5 +31,13 @@ public struct SQLDropTable: SQLExpression {
             }
         }
         self.table.serialize(to: &serializer)
+        if serializer.dialect.supportsDropBehaviour {
+            serializer.write(" ")
+            if let dropBehaviour = behaviour {
+                dropBehaviour.serialize(to: &serializer)
+            } else {
+                SQLDropBehaviour.cascade.serialize(to: &serializer)
```

Comment:
This makes the default behavior of code dropping tables `CASCADE`. Is that what we want?

Reply:
Yes, that’s right, I've changed the default to `RESTRICT`.